batch_seqr_loader/batch_workflow.py

```diff
@@ -1146,29 +1612,48 @@ def _add_final_gather_vcf_step(
     return j
 
 
-def can_reuse(fpath: str, overwrite: bool) -> bool:
-    """
-    Checks if the file `fpath` exists and we are not overwriting
-    """
-    if not file_exists(fpath):
-        return False
-    elif overwrite:
-        logger.info(f'File {fpath} exists and will be overwritten')
-        return False
-    else:
-        logger.info(f'Reusing existing {fpath}. Use --overwrite to overwrite')
-        return True
+def _verify_reads_data(
+    reads_data: Union[None, str, List[Union[str, List[str]]]]
+) -> Union[
+    str, Tuple[List[str], List[str]], None
+]:  # pylint: disable=too-many-return-statements
 
+    if not reads_data:
+        logger.error(f'     ERROR: no "reads" data')
+        return None
 
-def hash_sample_names(sample_names: Iterable[str]) -> str:
-    """
-    Return a unique hash string from a from a set of strings
-    :param sample_names: set of strings
-    :return: a string hash
-    """
-    for sn in sample_names:
-        assert ' ' not in sn, sn
-    return hashlib.sha224(' '.join(sorted(sample_names)).encode()).hexdigest()
+    error_msg = (
+        f'     ERROR: the "reads" meta data must be a list of '
+        f'str/list (for FASTQ), or a BAM or CRAM path. Got: {reads_data}'
+    )
+    if isinstance(reads_data, str):
```

Comment:
This isn't the format for reads in the sample metadata. They're formed CWL File objects, eg:

```json
"reads": [
    {
        "location": "gs://<bucket>/<sampleId>.bam",
        "basename": "<sampleId>.bam",
        "class": "File",
        "checksum": "md5:<md5-sum>",
        "size": 0, # in bytes
        "secondaryFiles": [
            {
                "location": "gs://<bucket>/<sampleId>.bam.bai",
                "basename": "<sampleId>.bam.bai",
                "class": "File",
                "checksum": null,
                "size": 0
            }
        ]
    }
],
"reads_type": "bam"
```

OR 

```json
"reads": [
    [
        {
            "location": "gs://<bucket>/{sampleId}_L001_R1.fastq.gz",
            "basename": "{sampleId}_L001_R1.fastq.gz",
            "class": "File",
            "checksum": "md5:checksum",
            "size": 13939740017
        },
        {
            "location": "gs://<bucket>/{sampleId}_L001_R2.fastq.gz",
            "basename": "{sampleId}_L001_R2.fastq.gz",
            "class": "File",
            "checksum": "md5:checksum",
            "size": 14710616940
        }
    ],
    [
        {
            "location": "gs://<bucket>/{sampleId}_L002_R1.fastq.gz",
            "basename": "{sampleId}_L002_R1.fastq.gz",
            "class": "File",
            "checksum": "md5:checksum",
            "size": 14061476086
        },
        {
            "location": "gs://<bucket>/{sampleId}_L002_R2.fastq.gz",
            "basename": "{sampleId}_L002_R2.fastq.gz",
            "class": "File",
            "checksum": "md5:checksum",
            "size": 14824459848
        }
    ]
],
"reads_type": "fastq"
```

Reply:
Yeah, it just worked for my fake test data, forgot to update it to the real entries! Did now :)